### PR TITLE
Fix CreateTopicsResponse generation

### PR DESCRIPTION
**Pull request by ivanyu**
The key is in handling optionality and defaults of primitive types correctly.

### CODE REVIEW

File: codegen/parser.py [hunk 1]
```diff
@@ -272,6 +272,18 @@ class PrimitiveField(_BaseField):
     default: str | int | float | bool | None
 
     def is_nullable(self, version: int) -> bool:
+        # Primitive types are never optional
```
Comment (aiven-anton) [inline codegen/parser.py:275]: thought (non-blocking): This is interesting, I wonder whether this means that we can theoretically treat the combination of one of those types together with `nullableVersions` as a parse error when parsing schema.

**Review by aiven-anton — COMMENTED** [review]
Just the UUID default needs adjusting IMO, otherwise ready to merge.

File: codegen/generate_schema.py [hunk 2]
```diff
@@ -183,6 +183,31 @@ def format_dataclass_field(
...
+    elif field_type is Primitive.int64:
+        return "i64(0)"
+    elif field_type is Primitive.uint16:
+        return "u16(0)"
+    elif field_type is Primitive.uint32:
+        return "u32(0)"
+    elif field_type is Primitive.uint64:
+        return "u64(0)"
+    elif field_type is Primitive.uuid:
+        return "uuid.UUID(int=0)"
```
[the remark this answers is not in the document]
Comment (ivanyu) [reply codegen/generate_schema.py]: Right, I failed to notice this. Fixed.

File: codegen/generate_schema.py [hunk 3]
```diff
@@ -183,6 +183,31 @@ def format_dataclass_field(
...
+    elif field_type is Primitive.float64:
+        return "f64(0.0)"
+    else:
+        return "None"
```
[the remark this answers is not in the document]
Comment (ivanyu) [reply codegen/generate_schema.py]: Thanks, applied

File: codegen/parser.py (hunk 1 above)
Comment (ivanyu) [reply codegen/parser.py:275]: Probably so. Kafka [does this](https://github.com/apache/kafka/blob/5b295293c06f80accb86df5c46f78c1e5629f86a/generator/src/main/java/org/apache/kafka/message/FieldSpec.java#L95) (`canBeNullable` if `false` for primitives).

File: codegen/generate_schema.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (ivanyu) [reply codegen/generate_schema.py]: Hm, fails on `mypy`

[the remark this answers is not in the document]
Comment (antonagestam) [reply codegen/generate_schema.py]: Ah sorry, this needs another conditional branch to handle the nullable primitives explicitly.

[the remark this answers is not in the document]
Comment (ivanyu) [reply codegen/generate_schema.py]: Does it make sense now?

**Review by aiven-anton — APPROVED** [review]

File: codegen/generate_schema.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (aiven-anton) [reply codegen/generate_schema.py]: Yes, thanks! 👌

**Pull request merged**